.prettierignore

```diff
@@ -8,6 +8,8 @@ packages/cli/templates/form-trigger.handlebars
 cypress/fixtures
 CHANGELOG.md
 .github/pull_request_template.md
+# Ignored for now
+**/*.md
```

Comment:
since we are ignoring this for now. can we also revert all markdown files as well for now?
Once that is done, this LGTM. But, I can't approve the PR because I created it.
Once the markdown files are reverted, please feel free to approve and merge this 🙏🏽